### PR TITLE
WordPress plugins: update the polylang cookie solution

### PR DESCRIPTION
**Pull request by jkudish**
## Summary

**[WordPress Plugins and Themes with Known Issues: Polylang](https://docs.pantheon.io/plugins-known-issues#polylang)** - This PR seeks to update the proposed solution to the Polylang's cookie preventing cache hits. The previous solution removed the polylang cache key. The new solution makes the cookie a cache-varying cookie which is a better and more reliable solution.

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by rachelwhitton** [comment]
@ryanshoover is there someone on your team you can assign to provide a technical peer review here?

**Review by theambert — APPROVED** [review]
These changes look good. This updated recommendation has been successfully implemented on a number of customer sites.

**Comment by stevector** [comment]
Thanks @theambert! I'll merge

File: source/content/plugins-known-issues.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+![CleanShot 2024-03-06 at 15 02 11@2x](https://github.com/pantheon-systems/documentation/assets/260253/cb95b35d-3dd4-4a24-89ad-c24316f79317)---
```
Comment (stevector) [inline source/content/plugins-known-issues.md:1]: @rachelwhitton should this screenshot be here?

**Review by stevector — CHANGES_REQUESTED** [review]
@rachelwhitton I think this screenshot should be removed, yes?

**Comment by stevector** [comment]
Assigning to @rachelwhitton to look at the screenshot

File: source/content/plugins-known-issues.md (hunk 1 above)
Comment (rachelwhitton) [reply source/content/plugins-known-issues.md]: Yeah, It's in the wrong spot, but also I don't think we need this. We don't show a diff for the provided code changes anywhere else in this doc, let's not do it here. I removed it from the doc

Comment (rachelwhitton) [reply source/content/plugins-known-issues.md]: @stevector can you update your review and merge when ready? This is good to go on my side

**Review by rachelwhitton — APPROVED** [review]

**Review by stevector — APPROVED** [review]

**Pull request merged**